frontend/src/components/core/Link.tsx

```diff
@@ -15,8 +23,29 @@ const createButtonLink = (Button:any, props:any) => {
   return <Link to={to}>{button}</Link>;
 };
 
+// Wrap a button inside of a link with an auto margin-left
+const createButtonLinkAutoLeftMargin = (Button:any, props:any) => {
+  const {
+    to,
+    children,
+    ...rest
+  } = props;
+  const button = (
+    <Button {...rest}>
+      {children}
+    </Button>
+  );
+
+  return <AutoLeftMarginLink to={to}>{button}</AutoLeftMarginLink>;
+};
+
 export const PrimaryButtonLink = (props:any) => createButtonLink(PrimaryButton, props);
 
+export const InvertedSmallButtonLinkAutoLeftMargin = (props: any) => createButtonLinkAutoLeftMargin(
```

Comment:
That's quite the mouthful lol

Reply:
Yeah it's such a specific element haha - not sure what to do about that one, I would style it with a more specific name directly in `ProblemDisplay.tsx`, but I have to create this wrapper with the link. We can change this name if needed, np 👍